contracts/0.8.25/vaults/dashboard/NodeOperatorFee.sol

```diff
@@ -222,7 +227,7 @@ contract NodeOperatorFee is Permissions {
         if (latestCorrectionTimestamp >= _lazyOracle().latestReportTimestamp()) revert CorrectionAfterReport();
 
         // If the vault is quarantined, the total value is reduced and may not reflect the exemption
```

Comment:
but we are now calculating fees over quarantined amount, can it be relaxed or better stay strict?